test/gaudi_opts/test_link_conversion_edm4hep.py

```diff
@@ -74,12 +86,15 @@
 )
 
 mcLinkConverter = EDM4hep2LcioTool("MCLinkConverterToEDM4hep")
-mcLinkConverter.convertAll = False
-mcLinkConverter.collNameMapping = {
-    "TrivialMCRecoLinks": "TrivialMCRecoLinks",
-    "MCParticles": "MCParticles",
-    "PseudoRecoParticles": "PseudoRecoParticles",
-}
+if args.use_gaudi_algorithm:
+    mcLinkConverter.convertAll = True
+else:
+    mcLinkConverter.convertAll = False
```

Comment:
Can we remove this differentiation here? I think I originally put it in effectively as an optimization. Or would this still require https://github.com/key4hep/k4FWCore/pull/290

Reply:
Yes, actually when checking I thought we have `convertAll` in very places if not none so maybe it's a good idea to have some more.

Reply:
Actually I reverted to how it was before. I found that when `convertAll` is set, from EDM4hep to LCIO it won't convert collections that are created since it only checks which names have been passed to the converter. This could be improved by checking in the store, and we already have some functionality to do that. I'll open an issue to discuss that. We can change to always convert all later.